src/CommunityToolkit.Maui.Core/Interfaces/IAlert.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Alert
+/// </summary>
+public interface IAlert : IAsyncDisposable
+{
+	/// <summary>
+	/// Message text
+	/// </summary>
+	string Text { get; }
+
+	/// <summary>
+	/// Dismiss the alert
+	/// </summary>
+	Task Dismiss(CancellationToken token = default);
```

Comment:
We don't have `async` calls on the implementation. We should change it to be `void` or `ValueTask` (if there's a chance of an async implementation). I know that you mentioned that this will break other APIs, but this is the time to break things and improve the API.

`Task` always allocates an object on the `heap`, even if we don't have an `async` call. With `ValueTask` we don't have that allocation if there's a hot path.